stable/fairwinds-insights/values.yaml

```diff
@@ -637,8 +637,8 @@ migrateHealthScoreJob:
 
 cronjobExecutor:
   image:
-    repository: bitnami/kubectl
-    tag: "1.22.8"
+    repository: rancher/kubectl
```

Comment:
What about alpine/kubectl?

Reply:
End-up using alpine due to rancher image being based on scratch